low-level/framework/utils/iem.py

```diff
@@ -174,3 +174,6 @@ def generate_iem(module, event_code, severity, description):
         except EventMessageError as e:
             logger.error("Failed to send IEM alert."
                          f"Error:{e}")
+        except Exception as err:
+            logger.error("Failed to send IEM alert."
+                         f"Error:{err}")
```

Comment:
I also don't get what is the point of having the special exception handling when we don't have distinct action for such a specific exception. @shriya-deshmukh, @cdeshmukh, @sandeepsgt ??

Reply:
In case of Kafka unavailability, while executing 'EventMessage.send()' it encouters 'kafkaerror' which is not getting handled in
EventMessageError  exception, hence added generic exception.

Reply:
yes, I meant what is the use of EventMessageError exception handling when the action for both Generic and specific is the same..